os/dbuild.sh

```diff
@@ -475,6 +487,19 @@ function BUILD()
 	UPDATE_STATUS
 }
 
+function TRAP_RUN()
+{
+	TRAPCMD=ramdumpParser.py
+	TRAPCMD+=" -t"
+	TRAPCMD+=" "$2
+
+	echo "Executing: "$TRAPCMD
+
+	printf "\nRunning TRAP tool..."
+	# execute TRAP script
+	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -it -w /root/tizenrt/tools/trap --privileged tizenrt/tizenrt:1.5.6 python3.7 $TRAPCMD 
```

Comment:
Is -it flag required? afaik it's for opening an interactive shell using docker. I think it's not required.

Reply:
Since we are executing another python script from within the parser, this is required to avoid early execution.